File: JavaScript/8js 2/mystery.js
/*
|--------------------------------------------------------------------------
| Return to Sanity
|--------------------------------------------------------------------------
|
*/

function mystery() {
   let res = { sanity: "Hello" };
   return res;
}


const mystery = () => ({ sanity: "Hello" });


function mystery() {
   return { sanity: "Hello" };
}


console.log(mystery(), { sanity: "Hello" }, "Mystery has not returned to sanity");

File: Python/8py 2/mystery.py
"""
|--------------------------------------------------------------------------
| Return to Sanity
|--------------------------------------------------------------------------
|
"""

def mystery():
   res = {'sanity': 'Hello'}
   return res


mystery = lambda: {'sanity': 'Hello'}


def mystery():
   return {'sanity': 'Hello'}


print(mystery(), {'sanity': 'Hello'}, 'Mystery has not returned to sanity')